src/biokbase/installed_clients/CatalogClient.py

```diff
@@ -1,3 +1,4 @@
+# -*- coding: utf-8 -*-
```

Comment:
the changes in this file are due to running `kb-sdk install catalog` - the client has been updated since whenever this file was last touched.